src/Uno.UI/UI/Xaml/Controls/ScrollBar/ScrollBar.cs

```diff
@@ -978,6 +981,27 @@ internal bool IsIgnoringUserInput
 			}
 		}
 
+		// Workaround for ScrollViewer. On Windows, ScrollViewer ignores ScrollBar's SmallChange/LargeChange values.
+		// No matter how SmallChange/LargeChange are set, ScrollViewer will always scroll by 16 (SmallChange) or
+		// ScrollViewer's Width/Height (depending on scrolling orientation).
+		private (double smallChange, double largeChange) GetAdjustedChange()
```

Comment:
Even if usually I like your approach, I think that in this specific case we should do the "opposite way" : instead of walking the tree up when we click the button, the `ScrollViewer` should update the `SmallChange` and `LargeChange` at the right time. This is how the `SV` works and it already does that for lot of properties, especially on its `ScrollContentPresenter`.

We have 2 methods for that [`UpdateComputedVerticalScrollability`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs#L803) and [`UpdateComputedHorizontalScrollability`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs#L840). They will be invoked each time a property that impacts the layouting of the `SV` or its layouting contraints changed.